### ISSUE TITLE
🐛 On docker-compose run, message "Delivery is not available in your country for some of the items of your cart." is nearly always displayed without legit motive

### ISSUE THREAD

**Issue by Tirodem** [B2Bitcoin/beBOP#1199]
![image](https://github.com/B2Bitcoin/beBOP/assets/50206014/a3ac5e10-9443-4b33-9331-865446cf6bf0)
tl;dr : on docker-compose run, I have this despite
- product not having physical component
![image](https://github.com/B2Bitcoin/beBOP/assets/50206014/14269aa0-7081-4712-8e0e-3612e090113c)
- delivery fees settings being flat with a global price for every "other countries"
![image](https://github.com/B2Bitcoin/beBOP/assets/50206014/fcae6e90-72f5-4ee7-8361-c17f85227c44)

It's also displayed on cart preview :
![image](https://github.com/B2Bitcoin/beBOP/assets/50206014/71df292d-3295-4971-a615-d6bb6137f5fc)

But nothing anymore on checkout as there's no delivery :
![image](https://github.com/B2Bitcoin/beBOP/assets/50206014/b1536ccb-d503-42e0-a631-59648b3d414c)

**Comment by coyotte508** [comment]
Do you have `git lfs` installed?

What's the result of `ls -lsah ./assets` ?

**Comment by Tirodem** [comment]
I'll check.
git-lfs is supposedly installed but training VPS was recycled for a pilot, I upped a new one tonight.
Can it be related to what we saw earlier with nginx forward header & IP check ? Does it work with docker-compose install & run on be-BOP ?

**Comment by coyotte508** [comment]
yes it may be related to nginx forward header & IP check, in that case setting the values in `.env` should be enough for docker

**Comment by coyotte508** [comment]
This is already in the readme though?

https://github.com/B2Bitcoin/beBOP/blob/29993bd8173ee1a89872266312bdc72daaac1f7c/README.md#L60-L80

Or maybe do you mean adding a "reverse proxy" section in the docker compose section of the README?

**Comment by Tirodem** [comment]
My bad, didn't see it. It's cool so 👌
